resources/landmarks/landmarks.json

```diff
@@ -0,0 +1,390 @@
+{
+  "independence": {
+    "name": "Independence, Missouri",
+    "id": 0,
+    "isCity": true,
+    "isRiver": false,
+    "altRoute": false,
+    "sayings": [
+      "first saying",
+      "second saying",
+      "third saying"
```

Comment:
HAHA this probably wasn't intended to be committed. The correct sayings should go here

Reply:
Doh!